SparkPost/2web2ui
# SA-1619 - (UI) Compare By - Aggregate section update

[SA-1619](https://sparkpost.atlassian.net/browse/SA-1619)

### What Changed

- Adds API helpers to use with `useSparkPostQuery` that map to existing Redux actions for `metrics` endpoints
- Expose relevant query params for the `getQueryFromOptionsV2` helper
- Conditionally render `CompareByAggregatedMetrics` component depending on whether there are active comparisons present in feature's state
- Moves constants to a single index file within the report builder feature
- Upgrades Review Badger

### How To Test
- Click on "Add Comparison" and add at least two comparisons
- Verify that below the charts, aggregated data renders
- Look at the network requests in dev tools and verify the correct requests are being made
- Verify that the section updates when metrics are added and removed
- Verify that the section renders when the user refreshes the page or pastes the URL in a new tab or browser window

### To Do
